core/app/models/spree/payment_create.rb

```diff
@@ -3,16 +3,24 @@
 module Spree
   # Service object for creating new payments on an Order
   class PaymentCreate
+    PAYMENT_NOT_PROVIDED = Object.new.freeze
+
     # @param order [Order] The order for the new payment
     # @param attributes [Hash,ActionController::Parameters] attributes which are assigned to the new payment
     #   * :payment_method_id Id of payment method used for this payment
```

Comment:
What about adding documentation for the new `id` attribute, required in case we want to use an existing payment?